assets/src/edit-story/components/panels/panel/panel.js

```diff
@@ -188,6 +193,15 @@ function Panel({
   const panelTitleId = `panel-title-${name}-${uuidv4()}`;
   const panelTitleReadable = `panel-title-${name}`;
 
+  useStoryTriggerListener(
+    expandOnTrigger,
+    useCallback(() => {
+      if (isCollapsed) {
+        expand(true);
+      }
+    }, [isCollapsed, expand])
+  );
+
```

Comment:
This hook happens in every panel now - so I added an event that's ignored: `STORY_EVENTS.NULL`

Reply:
This feels a little weird, like this should be controlled from the implementation of the panel not the shared panel component but there is similar functionality happening here and I get that there's no other way to really hook into this and have it not get wires crossed. That said, i think what you've done here is pretty clean!

Reply:
Yeah it's totally weird 😞. I was debating whether we could pass in a prop that would force the menu to open once but having something like `open`, `isOpen`, `expand`, `isExpanded` would be confusing since this is not a controlled component

All the hiding/showing state is kept inside this component. Lifting it out would be ideal, but that would be a huge task since this is used in so many places.

Reply:
Yea also thought the approach was a little funky, but couldn't think of a simpler way to do this.

The only idea I had seemed more complex and I wasn't in love with either: render a context provider in the panel that allows you to access some of the internal methods like a `setExpanded` method. Then hook into the story events from the panel's child to call the expanded method 🤷 

Also feels really funky tho and more of a lift than what's currently here. 🤔

Reply:
I suppose the only question I have about this implementation and it's possible restrictions (since doing this other ways requires a significant lift) is: is there a way to indicate a panel should _collapse_ - is there a way to make this into more of a toggle so we can control both expand and collapse?